### PR TITLE
Release/2021.12.0

### LINKED ISSUE #290: Improve setup docs
# Problem Description

External user testing has shown that the setup process needs more / better documentation:

> There was no clear guide on how to get RDN token, or what service Raiden team is recommending to the user. (I provided user with funds later, after he did some research )

> User did not have a clear understanding of what he should use for ETC_RPC parameter.
Documentation should recommend best possible options, like running an ETH node(with exact sync type), or Infura or some other service, with summarized tradeoffs for each if there is more than one option.
User decided to use Infura, after researching ETH_RPC.

- [x] Explain that the service registry script does automatically pick up where it left off in case a previous run is aborted.
- [x] Add mention of to be expected costs for registering.
  -   [x] https://github.com/raiden-network/raiden-services/pull/1125 
- [ ] Either fix https://github.com/raiden-network/raiden-synapse-modules/issues/8 or explain `SERVICE_REGISTRY` parameter.
  - https://github.com/raiden-network/raiden-synapse-modules/pull/19 
- [x] Mention who the intended audience for running an RSB is and what commitments are expected.
- [x] Prolong tx waiting time in register script and maybe increase gas price further

### PR DESCRIPTION
**Pull request by palango**
Resolves #290

### CODE REVIEW

**Comment by palango** [comment]
@ulope I updated the docs based in the internal mails. Please check that.

Also, we seem to have the same dockerhub problems as in the services here. Please update the secrets as well :D

File: BUILD_VERSIONS [hunk 1]
```diff
@@ -2,5 +2,5 @@
 export SYNAPSE_VERSION=v1.35.1
 export POSTGRES_VERSION=10
 export NGINX_VERSION=1.17
-export RAIDEN_VERSION=v2.0.0
+export RAIDEN_VERSION=v3.0.0rc8
```
Comment (ezdac) [inline BUILD_VERSIONS:5]: There is no docker image for that Raiden version on dockerhub. That's why your docker build CI steps are failing.
For some reason the Github workflow did not trigger the build of the `raidennetwork/raiden:v3.0.0rc8` image, while the newest PR was build under the `develop` tagged image

**Review by ezdac — APPROVED** [review]
Looks good! You're builds are failing though because of non-existance of the Raiden image on dockerhub

File: BUILD_VERSIONS (hunk 1 above)
Comment (ezdac) [reply BUILD_VERSIONS:5]: But in any case, shouldn't you first release raiden 3.0.0 and then release the new RSB with using the `v3.0.0` raiden version here?

Comment (palango) [reply BUILD_VERSIONS:5]: Can one retrigger those builds?

We just set up the docker image build after the release. So in worst case I create a new RC

File: README.md [hunk 2]
```diff
@@ -151,11 +159,11 @@ can identify this, if there is an `rcX` at the end of the version (E.g. `2019.03
 [latest full release](https://github.com/raiden-network/raiden-service-bundle/releases/latest). If the version is
 different from what you see below, you should stick to the "full release" and replace the version accordingly.
 
-1. Clone the [current release version of this repository](https://github.com/raiden-network/raiden-service-bundle/tree/2021.07.0rc0)
+1. Clone the [current release version of this repository](https://github.com/raiden-network/raiden-service-bundle/tree/2021.12.0rc0)
```
[the remark this answers is not in the document]
Comment (palango) [reply README.md:162]: Mh, the current link is git compatible, right? So from the release page you cannot clone the right state, right?!

File: BUILD_VERSIONS (hunk 1 above)
Comment (palango) [reply BUILD_VERSIONS]: Done

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ezdac) [reply README.md:162]: Yeah, true - for a git clone it's not directly compatible. Nevermind anyways

**Pull request merged**